preamble.tex

```diff
@@ -393,6 +393,9 @@
 \makecapperm*{G}{Global}
 
 \makeatletter
```

Comment:
Eh, I deliberately kept it in a separate \makeatletter/\makeatother pair as it's unrelated to the commands that follow but I guess this is ok for something that *should* be short-lived

Reply:
Hmm, I could do that, doesn't hurt.